### ISSUE TITLE
[Package Request] - fzf

### ISSUE THREAD

**Issue by seanorama** [amazonlinux/amazon-linux-2022#89]
**What package is missing from Amazon Linux 2022? Please describe and include package name.**
[fzf](https://github.com/junegunn/fzf) is a general-purpose command-line fuzzy finder.

It is available in most Linux distributions, including Fedora, from the default OS packages.

**Is this an update to existing package or new package request?**
New

**Is this package available in Amazon Linux 2?**
No

**Comment by reegnz** [comment]
Yes please! A fuzzy finder in todays world is almost as basic of a tool as grep IMHO.
